packages/edit-post/src/components/header/template-title/index.js

```diff
@@ -25,11 +25,18 @@ function TemplateTitle() {
 		return null;
 	}
 
+	let templateTitle = __( 'Default' );
+	if ( template && template?.title?.raw ) {
```

Comment:
I feel like making this inline with a triple `||` could be more straightforward, but nothing so important. If you keep it this way, loose the first `template` check.

Reply:
we have three cases so it's not really possible  to inline or at least not with a single condition which means it becomes harder to read for me.